CourseSketchProjects/CourseSketchDatabaseServer/src/test/java/database/institution/mongo/BankProblemManagerTest.java

```diff
@@ -122,9 +125,10 @@ public void insertBankProblemPermissions() throws Exception {
 
         String problemBankId = BankProblemManager.mongoInsertBankProblem(db, bankProblem.build());
 
-        final DBRef myDbRef = new DBRef(db, PROBLEM_BANK_COLLECTION, new ObjectId(problemBankId));
-        final DBObject mongoBankProblem = myDbRef.fetch();
+        final DBCollection bankProblemCollection = db.getCollection(getCollectionFromType(School.ItemType.BANK_PROBLEM));
+        final DBObject cursor = bankProblemCollection.findOne(convertStringToObjectId(problemBankId));
 
+        Assert.assertNotNull(cursor);
```

Comment:
Aren't Assert's mainly used for programmers while coding and should be removed when deploying?